lib/api/traversing.js

```diff
@@ -697,7 +697,7 @@ exports.not = function (match, container) {
     });
   }
 
-  return container._make(elements);
```

Comment:
This is using the same pattern as `filter` right now. `filter` can be called with `this` being an array. We don't do that anywhere for `not` afaict, but maintaining the same pattern should be good in any case.

Reply:
ok, reverted